src/AzureIoTHub.Portal.Server.Tests.Unit/Mappers/DeviceTwinMapperTests.cs

```diff
@@ -49,8 +49,18 @@ public void CreateDeviceDetails_StateUnderTest_ExpectedBehavior()
             };
 
             twin.Tags[nameof(DeviceDetails.ModelId).ToCamelCase()] = "000-000-001";
-            twin.Tags[nameof(DeviceDetails.LocationCode).ToCamelCase()] = Guid.NewGuid().ToString();
-            twin.Tags[nameof(DeviceDetails.AssetId).ToCamelCase()] = Guid.NewGuid().ToString();
+
+            Dictionary<string, string> customTags = new()
+            {
+                { "assetId", Guid.NewGuid().ToString() },
+                { "locationCode", Guid.NewGuid().ToString() }
+            };
+
+            foreach (KeyValuePair<string, string> customTag in customTags)
+            {
+                twin.Tags[customTag.Key.ToCamelCase()] = customTag.Value;
+            }
+
```

Comment:
Maybe a more elegant way to do that ? 
```suggestion
            twin.Tags["assetId"] = Guid.NewGuid().ToString();
            twin.Tags["locationCode"] = Guid.NewGuid().ToString();
```